src/dry-runner.js

```diff
@@ -0,0 +1,136 @@
+import childProcess from "child_process";
+import {Promise} from "es6-promise";
+import fileUtils from "./file-utils";
+import fs from "fs";
+import logger from "./logger";
+import mockfs from "mock-fs";
```

Comment:
Yeah that is correct. It doesn't actually replace anything until you call it, but I'll go ahead lazy require it to make sure we correctly read the package file.